### ISSUE TITLE
Bees going instantly back into hive

### ISSUE THREAD

**Issue by BIGF4TGG** [JDKDigital/productive-bees#294]
Hello I just noticed this but my bees in the end will come out for a split second and and go right back into the hive without gathering their resource whether it be flowers or blocks of material.

**Comment by JaisDK** [comment]
Which version of the mod are you on?

**Comment by BIGF4TGG** [comment]
Currently on ATM7 0.4.22

**Comment by JaisDK** [comment]
I think you just need to update to get it fixed

**Comment by BIGF4TGG** [comment]
okay thank you

**Comment by artdude543** [comment]
Hello,

We were noticing this issue with the ATM modpack also. We were on .23 of the pack (which at the time was using the mod version: 0.9.0.9) since when we moved the server/clients to pack version .30 (which is using the mod version: 0.9.1.1).

The bees are acting in an odd way as described by the OP where upon nightfall they are dropping any items, etc., then try to come out of the hive into the world. Before (assuming) the check for day/night occurs and they jump back into the hive. This is causing quite a bit of TPS hits due to the constant entity switching into/out of the world. This is affecting bees which are not even supposed to come out at night.

Using the simulator does act as a workaround to stop the bees from wanting to come out at night.

I've attached a video to give some visual context to the issue. [Video Link](https://www.youtube.com/watch?v=uLvLuqmkxKg)

Any help/feedback greatly appreciated. Also, love the mod has been amazing to watch my friend evolve his bee area and then leech off the goods 😂
